gui/multiSwitch.py

```diff
@@ -37,7 +37,7 @@ def handleDrag(self, type, info):
             if h:
                 h(type, info)
 
-    def AddPage(self, tabWnd=None, tabTitle=_("Empty Tab"), tabImage=None):
```

Comment:
Same as above, I think we need to translate the literal strings like this which are then passed into the other function as-is.

I'm wondering if you ran into a problem that required you to change it? I have done very little testing with the stuff that I've commited, so it's possible I'm not seeing the reasoning :)

Reply:
Yeah, It's a problem regarding default tab "Empty Tab". I have that literal string translated but the UI still showing the raw string.

Now I think about this, when does `_(...)` called? I think it's when we call `import gui.multiSwitch`, and at that time, we might not have locale correctly set up. and when the `gui.mainFrame` calls this with default args (see also `gui/mainFrame.py:270`), I assume it would bind to the string literal "Empty Tab" before translation could happen.

If you get rid of `_ = wx.GetTranslation` and use this snippet:
```py
def _(*args, **kwargs):
    return wx.GetTranslation(*args, **kwargs)
```
Then add a break point on the `return` line, you will see that it's a series of `import` calls, all the way back to `pyfa.py:140`.

Reply:
I just tested on a small test app, and you're right - when used as a default argument, the translation isn't done at runtime.

For now I think a decent workaround would be to set the default to `None` and then default it in the function itself:

```
def AddPage(self, tabWnd=None, tabTitle=None, tabImage=None):
    tabTitle = tabTitle or _("Empty Tab")
```

I'm going to make some inquiries on the wx forum to see if there's a better / standard way of dealing with this

Reply:
https://discuss.wxpython.org/t/translations-dont-happen-when-set-as-default-argument/34719 
https://github.com/wxWidgets/Phoenix/issues/1693

I have these two feelers out, it occurred to me ass I was making the issue that I've never attempted a function call as a default argument in python before, so it might just not be a thing.

But I'm good with using things like `tabTitle = tabTitle or _("Empty Tab")` as a workaround